### ISSUE TITLE
Add cornerplot function

### ISSUE THREAD

**Issue by cpfiffer** [TuringLang/MCMCChains.jl#26]
It would be nice to have a quick function that accepts a chain and a variable set and returns a corner plot. We currently do something like this in the [logistic regression tutorial](http://turing.ml/tutorials/2-logisticregression/) for Turing, but it was outside the library. 

In that case, we did something like this:

```julia
# The labels to use.
l = [:student, :balance, :income]

# Extract the parameters we want to plot.
w1 = chain[:student]
w2 = chain[:balance]
w3 = chain[:income]

# Show the corner plot.
cornerplot(hcat(w1, w2, w3), compact=true, labels = l)
```

It would be nice to be able to call `cornerplot(chain, [:var1, :var2, :var3])` and get a plot back. @mkborregaard would you be willing to help on this one?

**Comment by mkborregaard** [comment]
sure :-)

**Comment by mkborregaard** [comment]
In this case it's actually easist to just add a new method on cornerplot.

**Comment by mkborregaard** [comment]
Slack discussion ended up deciding to make a `chainsummaryplot` recipe here to do this

**Comment by mkborregaard** [comment]
I know this contradicts my previous advice about keeping the Plots' names. 
The reason is that you want to dispatch to a method from StatPlots while still only depending on RecipesBase. That's tricky, so in this case it's better to come with your own recipe. 
Another more obvious name could be `chain_cornerplot`?

**Comment by mkborregaard** [comment]
Wait, thinking more about it, i think we should integrate it in the framework I added here: https://github.com/TuringLang/MCMCChain.jl/pull/19
So, `plot(chain, syms, colordim = :chain, series = :cornerplot)`?

**Comment by cpfiffer** [comment]
Closed by #19.
